### PR TITLE
Add meta descriptions to blog posts that have not been updated

### PR DESCRIPTION
**Pull request by bermudezmt**
Signed-off-by: bermudezmt <bermudez.mt@gmail.com>

### Proposed changes

This covers 1-10 blog posts on your list. Will try to hit the rest later today.

### Related issues (optional)

#668

### CODE REVIEW

File: content/blog/pulumi-a-better-way-to-kubernetes/index.md [hunk 1]
```diff
@@ -3,6 +3,7 @@ title: "Pulumi: A Better Way to Kubernetes"
 authors: ["levi-blackstone"]
 tags: ["Kubernetes"]
 date: "2019-05-21"
+meta_desc: "Pulumi makes Kubernetes more accessible using familiar languages and your favorite tools. Create managed Kubernetes clusters, manage any Kubernetes resource incluindg CRDs and related CustomResources, import Helm charts and manifests, and mix and match these resources within your Pulumi program."
```
Comment (ericrudder) [inline content/blog/pulumi-a-better-way-to-kubernetes/index.md]:
```suggestion
meta_desc: "Pulumi makes Kubernetes more accessible using familiar languages and your favorite tools. Create managed Kubernetes clusters, manage Kubernetes resources, including CRDs and related CustomResources, import Helm charts and manifests, and mix and match these resources within your Pulumi program."
```

File: content/blog/pulumi-heart-google-cloud-platform/index.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ title: "Getting Started on Google Cloud Platform with Pulumi"
 authors: ["luke-hoban"]
 tags: ["Serverless","Kubernetes","Features","CI/CD","GCP"]
 date: "2019-04-09"
-
+meta_desc: "Pulumi offers tooling that works with GCP's rich features for easy collaboration, sharing, and reuse. Pulumi gives you full access to the full Google Cloud Platform, including working with Kubernetes and GKE, super simple serverless with Google Cloud Functions and continuous delivery with Google Cloud Build, and managing deployments with the Pulumi Console."
```
Comment (ericrudder) [inline content/blog/pulumi-heart-google-cloud-platform/index.md]:
```suggestion
meta_desc: "Pulumi offers tooling that works with GCP and enables collaboration, sharing, and reuse. Pulumi gives you full access to the full Google Cloud Platform, including Kubernetes and GKE, super simple serverless with Google Cloud Functions, continuous delivery with Google Cloud Build, and managing deployments with the Pulumi Console."
```

File: content/blog/pulumi-meetup-recap-apis-custom-resources-and-github-webhooks/index.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ title: "Pulumi Meetup Recap: APIs, Custom Resources and GitHub Webhooks"
 authors: ["aydrian-howard"]
 tags: ["events"]
 date: "2019-07-16"
-
+meta_desc: "Pulumi HQ's July meetup featured talks from our two engineers: Paul who went over how programming languages help in building the best infrastructure code by writing a simple Pulumi program to create an RDS instance, and Matt who demoed a small bot that enforced pull request policies using GitHub Webhooks and Pulumi."
```
Comment (ericrudder) [inline content/blog/pulumi-meetup-recap-apis-custom-resources-and-github-webhooks/index.md]:
```suggestion
meta_desc: "Pulumi's July meetup featured talks from two key engineers: Paul reviewed how programming languages help in building the best infrastructure code and wrote a simple Pulumi program to create an RDS instance, and Matt demoed a nifty little bot that enforced pull request policies using GitHub Webhooks and Pulumi."
```

File: content/blog/reusable-cicd-components-with-circleci-orbs-for-pulumi/index.md [hunk 4]
```diff
@@ -3,7 +3,7 @@ title: "Reusable CI/CD components with CircleCI Orbs for Pulumi"
 authors: ["chris-smith"]
 tags: ["CI/CD"]
 date: "2018-11-07"
-
+meta_desc: "Pulumi is proud to be a CircleCI technology partner and we are continuing to make Pulumi the best tool for the continuous deployment part of CI/CD workflows. This blog post showcases Pulumi orbs in a CircleCI configuration using a simple job to build and update a JavaScript-based stack."
```
Comment (ericrudder) [inline content/blog/reusable-cicd-components-with-circleci-orbs-for-pulumi/index.md]:
```suggestion
meta_desc: "Pulumi is proud to be a CircleCI technology partner as we continue to make Pulumi the best tool for CI/CD workflows. This blog post showcases Pulumi Orbs with CircleCI, using a simple job to build and update a JavaScript-based stack."
```

File: content/blog/running-a-serverles-nodejs-http-server-on-aws-and-azure/index.md [hunk 5]
```diff
@@ -3,6 +3,8 @@ title: "Running a Serverless Node.js HTTP Server on AWS and Azure"
 authors: ["cyrus-najmabadi"]
 tags: ["Serverless"]
 date: "2018-10-02"
+meta_desc: "Pulumi introduces the cloud.HttpServer API which makes it easy to serve a standard Node.js HTTP server as a serverless API on any cloud platform. The API is designed to work well with the existing large middleware
+ecosystem and critically, the same HttpServer API can be implemented consistently on AWS, Azure and GCP - so you can write once and deploy to any cloud."
```
Comment (ericrudder) [inline content/blog/running-a-serverles-nodejs-http-server-on-aws-and-azure/index.md]: it looks like there might be an extra CR here, however, the display looks right to me???

Comment (bermudezmt) [reply content/blog/running-a-serverles-nodejs-http-server-on-aws-and-azure/index.md]: What do you mean by CR? :) It does look right to me, too.

**Comment by ericrudder** [comment]
carriage return ... (you know, that little lever that moves the paper up in
your typewriter)

**Pull request merged**